packages/example-getting-started/docs/6-repository.md

```diff
@@ -1,15 +1,22 @@
 ### Create your repository
 
+The repository pattern is one of the more fundamental differences between
+LoopBack 3 and 4. In 3, you would use the model class definitions themselves
+to perform CRUD operations. In LoopBack 4, the layer responsible for this has
+been separated from the definition of the model itself, into the repository
+layer.
+
 Create another folder in `src` called `repositories` and inside of that folder,
 create two files:
 - `index.ts` (our export helper)
 - `todo.repository.ts`
 
 Our TodoRepository will contain a small base class that uses the
-`DefaultCrudRepository` class from `@loopback/repository` and will define the
-model type we're working with, as well as its ID type. We'll also inject our
-datasource so that this repository can connect to it when executing data
-operations.
+`DefaultCrudRepository` class from
```

Comment:
Maybe it could be inferred from the name of the class, but I think we should mention that `DefualtCrudRepository` provides the CRUD operations for you.